packages/itwinui-css/src/slider/slider.scss

```diff
@@ -215,8 +219,9 @@ $tick-height: $iui-baseline; // 12px
 
   .iui-slider-min,
   .iui-slider-max {
-    margin-left: $iui-xxs;
+    margin-left: calc($iui-xxs + $iui-sm);
```

Comment:
Combined the `margin-left` of the inner svg tag into line 222 itself. This was the only way to make it work.

What I wanted to do (to make it similar to the hierarchy structure of the horizontal slider) (lines 220-225):

```scss
.iui-slider-min,
  .iui-slider-max {
    margin-left: $iui-xxs;
    text-align: center;
    transform: translate(-50%, 0);

    svg {
      margin: $iui-sm;
    }
  }
```

What I ended up having to do to make it work:

```scss
.iui-slider-min,
  .iui-slider-max {
    margin-left: calc($iui-xxs + $iui-sm);
    text-align: center;
    transform: translate(-50%, 0);
  }
```